modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/SchemaManager.java

```diff
@@ -129,7 +128,7 @@
     private final ConcurrentMap<QueryTable, GridH2Table> dataTables = new ConcurrentHashMap<>();
 
     /** System VIEW collection. */
-    private final Set<SqlSystemView> systemViews = new GridConcurrentHashSet<>();
+    private final Map<String, SqlSystemView> sysViews = new ConcurrentHashMap<>();
```

Comment:
As far as I understand these changes related to support of dropping system view.
We shouldn't support dropping the system view.

Let's revert it and return empty data in case system view temporary can't return a real one.